server/authorise.test.js

```diff
@@ -0,0 +1,60 @@
+const request = require("supertest");
+const app = require("./app");
+const admin = require("firebase-admin");
+const authorise = require("./auth");
+const token = "MyToken";
+
+const validBearerToken = {
+    token: `Bearer ${token}`,
+};
+
+const noBearerPrefixToken = {
+    token: token,
+};
+
+const {
+    shortPasswordErrorMessage,
+    mockToken,
+    mockUid,
+    invalidEmailErrorMessage,
+    UnauthorizedErrorMessage,
+} = require("./test_utils/mocks/mockFirebase");
+
+jest.mock("firebase-admin", () => {
+    const { mockFirebase } = require("./test_utils/mocks/mockFirebase");
+    return mockFirebase;
+});
+
+describe("authorisation tests  ", () => {
+    beforeAll(() => {
+        //Used this originally for postman testing does not work without this line
+        app.post("/auth", authorise);
```

Comment:
/auth isn't an endpoint but I used it to send the token in the test(test won't work without this line)
I don't know if this is a fine method to do so